airflow/jobs/backfill_job.py

```diff
@@ -18,6 +18,7 @@
 from __future__ import annotations
 
 import time
+from concurrent.futures import Executor, ThreadPoolExecutor
```

Comment:
```suggestion
from concurrent.futures import Executor as FutureExecutor, ThreadPoolExecutor
```

(and matching else where: to avoid confusing with Airflow's own Executor class

Reply:
Removed